src/scss/includes/placeIcon.scss

```diff
@@ -8,9 +8,4 @@
   align-items: center;
   justify-content: center;
   flex-shrink: 0;
-
-  // indicates it's using the icomoon font
-  &:not(.icon) {
-    font-size: 20px;
-  }
```

Comment:
A similar css rule exists in "vehicleSelector.scss" and could be removed. 
(The vehicle icons are slightly displaced to the top).
https://github.com/Qwant/erdapfel/blob/2d20d7a0449b42ad36bfc0aed0487e1962bc3630/src/scss/includes/vehicleSelector.scss#L28